.env

```diff
@@ -0,0 +1,2 @@
+MONGO_VERSION=3.2
```

Comment:
We don't want to expose mongo version decision to users. Eg. it's only and only 3.2
So it's better to hardcode it in `docker-compose.yml`